cockatrice/src/window_main.cpp

```diff
@@ -795,3 +799,8 @@ void MainWindow::refreshShortcuts()
     aExit->setShortcuts(settingsCache->shortcuts().getShortcut("MainWindow/aExit"));
     aCheckCardUpdates->setShortcuts(settingsCache->shortcuts().getShortcut("MainWindow/aCheckCardUpdates"));
 }
+
+void MainWindow::notifyUserAboutUpdate()
+{
+    QMessageBox::information(this, tr("Information"), tr("A new version of the cockatrice client is available for download."));
```

Comment:
Right, this will address my initial concerns and avoid the possible security problems.